torq.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash 
 
-. ./setenv.sh                                                                                       # load the environment
+SETENV=setenv.sh                                                                                    # set the environment
```

Comment:
maybe this should check for an env var `SETENV` first & if it's set use that value? So someone could in theory do something like

```
$ SETENV=/home/user/setenv.sh start.sh all
```

or whatever to start up using a custom located env file without having to modify the start script. Obviously default would still be `setenv.sh` within current dir